### PR TITLE
Admin cancel #99

### LINKED ISSUE #99: User Story 36, Admin cancels a user's order
```
As an admin user
When I visit a user's order show page
If the order is still "pending", I see a button or link to cancel the order
When I click the cancel button for an order
The same behaviors happen as if the user canceled the order themselves
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #99 

* Adds `Admin::OrdersController`
  * Has destroy route
  * Adds route to config/routes
    * orders resource under admin namespace

* Adds `.cancel` to `Order` to dry up code (changes order status, saves, and calls `OrderItem` methods)
  * spec for that

*  Adds to admin/user show page 
  * Has some user info and the user's orders info (order: id, status, link to cancel if pending)

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -19,6 +19,14 @@ def self.any_complete?
     where(status: "complete").count > 0
   end
 
+  def self.cancel(order_id)
+    order = self.find(order_id)
+    order.status = "cancelled"
+    order.save
+    OrderItem.return_inventory_for(order.id)
+    OrderItem.unfulfill_items_for(order.id)
+  end
```
Comment (le3ah) [inline app/models/order.rb]: great refactor for the cancel method!  ❤️ fat models!

Comment (tonyr729) [reply app/models/order.rb]: +1
